packages/axelar-wasm-std/src/address.rs

```diff
@@ -223,6 +237,59 @@ mod tests {
         );
     }
 
+    #[test]
+    fn validate_solana_address() {
+        use crate::{address, assert_err_contains};
+        // Valid Solana address
+        let addr = "4f3J7t1HgX1t36k6rph2pYJrWxk9uT1RrB2K3nVHDh8D";
```

Comment:
add a case for all 1s, i.e. zero bytes address

Reply:
Added in 31d3abef94fa81ac4d42e4b967597388df129347